src/types.ts

```diff
@@ -268,6 +299,12 @@ export interface ExcalidrawProps {
       nativeEvent: MouseEvent | React.PointerEvent<HTMLCanvasElement>;
     }>,
   ) => void;
+  renderCustomElementWidget?: (appState: AppState) => void;
```

Comment:
let's put the UI rendering functions into `customElementsConfig[].renderButton` instead.

Reply:
we can but that would mean that the order of rendering will not be preserved since `customElementsConfig` is now an object, so should we add an `index` field to preserve order?